spec/controllers/challenge_claims_controller_spec.rb

```diff
@@ -20,40 +23,39 @@
 
     it 'will not allow you to see someone elses claims' do
       second_user = create(:user)
-      fake_login_known_user(user)
-      get :index, user_id: second_user.login
+      get :index, id: claim.id, user_id: second_user.id
       it_redirects_to_with_error(root_path, \
                                  "You aren't allowed to see that user's claims.")
     end
   end
 
   describe 'show' do
-    it 'redirects logged in user to the prompt' do
-      request_prompt = create(:prompt, collection_id: collection.id, challenge_signup_id: signup.id)
-      claim_with_prompt = create(:challenge_claim, collection: collection, request_prompt_id: request_prompt.id)
-      fake_login_known_user(user)
-      get :show, id: claim_with_prompt.id, collection_id: collection.name
-      it_redirects_to(collection_prompt_path(collection, claim_with_prompt.request_prompt))
+    it 'redirects' do
+      get :show, id: claim.id, collection_id: collection.name
+      it_redirects_to(root_path)
+    end
+
+    it 'needs a claim' do
```

Comment:
Descriptions need to describe how the code behaves in given contexts. You test that the code does `x` when it has a valid claim parameter and `y` when it doesn't have a valid claim parameter, not that it needs a claim. 

Also, I'd suggest explicitly setting an invalid claim and a valid collection here, just to be absolutely clear what you're testing: `get :show, claim_id: 999_999, collection_id: collection`. If for some reason `collection` doesn't work there, it's okay to leave it as an invalid collection (I think the phrase you're looking for in your tests is "nonexistent [thing]," btw, not "none existent [thing].")